2_process/src/compare_to_historic.R

```diff
@@ -1,5 +1,5 @@
 
-compare_to_historic <- function(target_name, historic_quantile_fn, current_data_fn) {
+compare_to_historic <- function(target_name, historic_quantile_fn, current_data_fn, inverse_sites) {
   
   current_data <- read_csv(current_data_fn, col_types = 'cDn')
```

Comment:
should this be `col_types = 'cDn'`?

Reply:
I don't think so ... I am pretty sure that the data has `site_no`, `Date`, and then `GWL` as the columns.

Reply:
sorry made a mistake, this comment was meant for `historic_quantiles` that has `site_no`, `quantile_nm`, `quantile_va`.

Reply:
ahhhh I see now. I did have that as `cnn` at one time, but changed to `ccn` because the `nm` in `quantile_nm` was intended to be read as "name" of the quantile (not "number") and it felt silly to have something called "name" but be numeric. I can see now that my decision likely has downstream consequences when `quantile_nm` is used in `approx`. I will change!